### DISCUSSION TITLE
Tira error al ejecutar el comando mvn clean:clean y no me crea proyecto al ejecutar mvn eclipse:eclipse

### DISCUSSION THREAD

**Discussion by juanmpereira** [dds-jv/foro#98] [Consultas de herramientas] [answered]
Buenas tardes, instale maven y lo probe con el comando mvn --version y me devuelve lo esperado. Ahora quiero crear un proyecto maven con lo cual me creo una carpeta prueba en documentos y dentro de ella creo un documento de texto con el nombre pom.xml . Alli copio y pego lo propuesto en el documento de maven cambiando los nombres a prueba del group id y artefact id. Por ultimo en el comando voy a esa direccion y pongo mvn clean:clean y me tira errores y lo mismo al querer hacer mvn eclipse:eclipse y no me crea el proyecto. Adjunto imagen de los erroes. Gracias y espero su respuesta. 
![WhatsApp Image 2020-04-02 at 15 37 09](https://user-images.githubusercontent.com/49444989/78288483-e111b680-74f7-11ea-90e9-3328a2f118d4.jpeg)

**Comment by gastonprieto** [comment]
Hola!

Antes que nada te comento que edite el issue para que se vea en el cuerpo, recorda que lo primero que aparece es una descripción de cierta información útil para tratar de documentar bien el problema. Pero ahí deberías borrarlo y completar con la duda que tengas.

En el caso que tengas un error ejecutándolo en la consola, conviene que copies el comando que ejecutaste y el error que te tira, pero en texto, así si alguien tiene el mismo error lo puede encontrar mas fácil. Y así tampoco chusmeamos tu casa por el reflejo 😉

**Comment by gastonprieto** [comment]
Sobre el error, el comando tenes que ejecutarlo en una carpeta que tengan un pom.xml, tipicamente es la carpeta base de tu proyecto (la carpeta inicial que subís al repositorio), puede ser que te este faltando ese archivo?

**Comment by juanmpereira** [comment]
Para crear el pom.xml uso algun editor en particular o lo puedo hacer en bloc de notas? Por que yo lo cree en el bloc de notas, tal como dice en el instructivo de youtube, agregandole las lineas para la version 8 que estando en el documento de maven que nos enviaron.

**Comment by gastonprieto** [comment]
Con un editor de texto alcanza. Nos pones el resultado de un `ls` en linux o un `dir` en windows, donde ejecutaste el comando de maven

**Comment by juanmpereira** [comment]
disculpa no entendi lo que me pedis, es el resultado de ejecutar el comando? lo paso aca.

```
C:\Users\juanm>cd C:\Users\juanm\Documents\prueba

C:\Users\juanm\Documents\prueba>mvn clean:clean
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------< org.apache.maven:standalone-pom >-------------------
[INFO] Building Maven Stub Project (No POM) 1
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.212 s
[INFO] Finished at: 2020-04-04T01:32:15-03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-clean-plugin:2.5:clean (default-cli): Goal requires a project to execute but there is no POM in this directory (C:\Users\juanm\Documents\prueba). Please verify you invoked Maven from the correct directory. -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MissingProjectException
```

**Comment by gastonprieto** [comment]
De la misma forma que ejecutas el comando mvn. Hay un comando para listar los archivos del directorio en el que estas parado. En windows el comando es `dir` en linux, el comando es `ls`. 

Si ese comando no te muestra un pom.xml es que estas ejecutando el comando de maven en el lugar incorrecto.

Que parece ser eso porque el error dice:
There is not found POM in this directory

**Comment by juanmpereira** [comment]
Este es el resultado de la ejecucion del comando dir
```
04/04/2020  02:01    <DIR>          .
04/04/2020  02:01    <DIR>          ..
02/04/2020  15:18               460 pom.xml.txt
               1 archivos            460 bytes
               2 dirs  42.558.181.376 bytes libres
```
por lo que veo,el archivo debe quedar como .xml unicamente y no en tipo de documento de texto como esta ahi?

**Comment by juanmpereira** [comment]
De ser asi el error al ejecutar el comando cambia en su descripcion a lo siguiente.
```
C:\prueba>mvn clean:Clean
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.202 s
[INFO] Finished at: 2020-04-04T02:10:49-03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Could not find goal 'Clean' in plugin org.apache.maven.plugins:maven-clean-plugin:2.5 among available goals clean, help -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoNotFoundException
```

**Comment by gastonprieto** [comment, marked as answer]
Genial, entonces el error es que la extensión esta mal, en vez de `pom.xml.txt` es `pom.xml`. 

El otro erro que pones mas abajo es porque escribiste `Clean` no `clean`, es sensible a mayusculas.

**Comment by juanmpereira** [comment]
Perfecto, ya me funciona . Muchas gracias por la ayuda!
